code/controllers/subsystem/triumph/triumph_adjust_procs.dm

```diff
@@ -8,6 +8,11 @@
 	if(!ismob(key_holder) && !ismind(key_holder) && !isclient(key_holder))
 		return
 	var/key = key_holder:key
+
+	// patreon triumph increase
+	if(key_is_donator(key))
+		amount *= 1.5
```

Comment:
Few cases to watch out for - if they refund any triumph buy (or are automatically refunded in case of any species failed roll), they will get 50% more than they spent, so they could use it to get infinite number of triumphs, same for communal triumph buys that refund automatically if they fail to reach their target.

Probably also worth it patching it for admins giving triumphs manually, so they are not surprised.